Code/GraphMol/FileParsers/test_data/atropisomers/AtropManyChiralsEnhanced2.sdf.expected2.sdf

```diff
@@ -49,7 +49,7 @@ M  V30 19 1 17 19 CFG=1
 M  V30 20 2 6 20
 M  V30 END BOND
 M  V30 BEGIN COLLECTION
-M  V30 MDLV30/STEABS ATOMS=(3 10 14 7)
```

Comment:
The original is correct - there should be 3 entries for the first stereo group. The 3rd one is for the atropisomer.
V30 MDLV30/STEABS ATOMS=(3 10 14 7)